handlers/slog/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package sloghandler provides a configurable logging handler for the Slog library, allowing users
+// to customize various aspects of log handling, including output destination, log level, attribute
+// replacement, and source information addition.
+package sloghandler
```

Comment:
I would add an example of the slog initialization with this `sloghandler`.

Reply:
Make sense, thank you